reark/src/test/java/io/reark/reark/pojo/IdentifiablePojoTest.java

```diff
@@ -0,0 +1,63 @@
+package io.reark.reark.pojo;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by apoi on 27/12/15.
+ */
+public class IdentifiablePojoTest {
+
+    private class TestPojo extends IdentifiablePojo<TestPojo> {
```

Comment:
maybe lets move the class to the end of the IdentifiablePojoTest. Will clutter less the code